Modules/Loadable/Terminologies/Logic/vtkSlicerTerminologiesModuleLogic.cxx

```diff
@@ -1519,6 +1519,12 @@ bool vtkSlicerTerminologiesModuleLogic::GetCategoryInTerminology(std::string ter
   rapidjson::Value& categoryObject = this->Internal->GetCategoryInTerminology(terminologyName, categoryId);
   if (categoryObject.IsNull())
   {
+    if (!categoryId.CodingSchemeDesignator.compare("SRT"))
```

Comment:
See the description of the problem below. I mark this as resolved because this line is the only content of this PR, and again the error is not about not finding certain terminology. The problem is that a found terminology contains very different data.